package.json

```diff
@@ -5,7 +5,7 @@
   "main": "index.js",
   "dependencies": {
     "@hyperswarm/hypersign": "^2.0.0",
-    "dht-rpc": "^4.1.6",
+    "dht-rpc": "git://github.com/davidmarkclements/dht-rpc.git#expose-rtt",
```

Comment:
note: temporary, needs to be altered to new release of dht-rpc before merge of this PR, waiting for merge and release of https://github.com/mafintosh/dht-rpc/pull/13